networking/ptp/ptp-events-rest-api-reference-v2.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ptp-events-rest-api-reference-v2"]
+= PTP events REST API v2 reference
+include::_attributes/common-attributes.adoc[]
+:context: using-ptp-hardware-fast-events-framework-v2
+
+toc::[]
+
+Use the Precision Time Protocol (PTP) fast event REST API v2 to subscribe cluster applications to PTP events generated by the bare-metal cluster node.
+
+The following v2 API endpoints are available:
+
+* xref:../../networking/ptp/ptp-events-rest-api-reference-v2.adoc#api-ocloud-notifications-v2-subscriptions_{context}[`api/ocloudNotifications/v2/subscriptions`]
+** `POST`: Creates a new subscription
+** `GET`: Retrieves a list of subscriptions
+** `DELETE`: Deletes all subscriptions
+
+* xref:../../networking/ptp/ptp-events-rest-api-reference-v2.adoc#api-ocloud-notifications-v2-subscriptions-subscription_id_{context}[`api/ocloudNotifications/v2/subscriptions/{subscription_id}`]
+** `GET`: Returns details for the specified subscription ID
+** `DELETE`: Deletes the subscription associated with the specified subscription ID
+
+* xref:../../networking/ptp/ptp-events-rest-api-reference-v2.adoc#api-ocloudnotifications-v2-health_{context}[`api/ocloudNotifications/v2/health`]
+** `GET`: Returns the health status of `ocloudNotifications` API
+
+* xref:../../networking/ptp/ptp-events-rest-api-reference-v2.adoc#api-ocloudnotifications-v2-publishers_{context}[`api/ocloudNotifications/v2/publishers`]
+** `GET`: Returns an array of `os-clock-sync-state`, `ptp-clock-class-change`, `lock-state`, and `gnss-sync-status` messages for the cluster node
+
+* xref:../../networking/ptp/ptp-events-rest-api-reference-v2.adoc#resource-address-current-state-v2_{context}[`api/ocloudnotifications/v2/{resource_address}/CurrentState`]
+** `GET`: Returns the current state of one the following event types: `os-clock-sync-state`, `ptp-clock-class-change`, `lock-state`, or `gnss-state-change` events
```

Comment:
Returns the current state of the event type specified by the {resouce_address}.  There could be more event types added in the future so better keep it generic.